empty/.post-create

```diff
@@ -0,0 +1,25 @@
+To begin development, install the Temporal CLI:
+
+  Mac: {cyan brew install temporal}
+  Other: Download and extract the latest release from https://github.com/temporalio/cli/releases/latest
+
+Start Temporal Server:
+
+  {cyan temporal server start-dev}
+
+Use Node version 16+:
+
+  Mac: {cyan brew install node@16}
+  Other: https://nodejs.org/en/download/
+
+Then, in the project directory:
+
+* Add your Activity Definitions to `src/activities.ts`.
+* Add your Workflow Definitions to `src/workflows.ts`.
+* Modify the `src/client.ts` file and replace `YOUR_WORKFLOW` with the name of your Workflow.
+* Optionally, add activity and tests to the `src/mocha` directory in files with the extension `.test.ts`.
```

Comment:
Maybe "Add Activity and Workflow tests"? This statement you have makes it sound like to add Activities to that directory